taglib/wavpack/wavpackproperties.h

```diff
@@ -139,6 +139,8 @@ namespace TagLib {
 
       void read(File *file, long streamLength);
       unsigned int seekFinalIndex(File *file, long streamLength);
+      int getNonStandardRate(unsigned char const *buffer, int bcount);
+      int getDsdRateShifter(unsigned char const *buffer, int bcount);
```

Comment:
These two methods to not access other members of the class. I would make them local to the *wavpackproperties.cpp* file and put them into a `namespace {}` which has to be below the block with the `#define`.